packages/app-store/basecamp3/config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Basecamp3",
+  "slug": "basecamp3",
+  "type": "basecamp3_other_calendar",
+  "logo": "logo.svg",
```

Comment:
NIT: Is there a way you can find a logo that could look OK in both light and dark modes?

Reply:
Seems like we're only allowed to use logos and screenshots from [here](https://basecamp.com/press). That being said, I think it depends on the theme as it looks fine in light mode.
![image](https://github.com/calcom/cal.com/assets/55296387/e111aad4-b5e2-47b4-b626-bf833eee7a2b)
Any idea on how to solve this? Should I change the background colour on the SVG?